spec/dummy/config/initializers/react_on_rails.rb

```diff
@@ -18,4 +18,7 @@
   # Server rendering only (not for render_component helper)
   config.server_renderer_pool_size  = 1   # increase if you're on JRuby
   config.server_renderer_timeout    = 20  # seconds
+
+  # Default is false, enable if your content security policy doesn't include `style-src: 'unsafe-inline'`
+  config.skip_display_none = false
```

Comment:
If you mean the generator template for the react_on_rails config, it's added in there with this PR.

https://github.com/shakacode/react_on_rails/blob/TangRufus-add-skip-display-none/lib/generators/react_on_rails/templates/base/base/config/initializers/react_on_rails.rb